codechecker_common/review_status_handler.py

```diff
@@ -316,15 +317,22 @@ def get_review_status_from_source(
         logic where we take the first comment into account in case of ambiguity
         or return "unreviewed" with a warning, like changed files.
         """
-        # The original file path is needed here, not the trimmed, because
-        # the source files are extracted as the original file path.
-        # TODO: Should original_path be strip('/') at store?
-        source_file_name = os.path.realpath(os.path.join(
-            self.__source_root, report.file.original_path))
+        if self.__source_root:
+            # sanitize paths like mass store run
+            source_file_name = path_for_fake_root(
+                report.file.original_path, self.__source_root)
+        else:
+            # in this branch we are operating on the original filesystem,
+            # and we need the path to he original file
```

Comment:
```suggestion
            # and we need the path to the original file
```

Reply:
"On this branch the server runs on the same file system where the source file is"

Reply:
Its very interesting and not at all trivial why we need to branch on this -- can you create an issue to remind us to document this properly?

Reply:
Thanks, for the review, will make an issue tomorrow